js/interface.js

```diff
@@ -236,19 +236,27 @@ var DynamicLists = (function() {
           var type = $(this).data('field');
           var $selector = $(this).parents('.filter-panel');
 
-          if (type === 'field') {
-            $selector.find('.panel-title-text .column').html(value === 'none' ? '(Field)' : value);
-          }
+          switch (type) {
+            case 'field':
+              $selector.find('.panel-title-text .column').html(value === 'none' ? '(Field)' : value);
+              break;
 
-          if (type === 'logic') {
-            var hideValueFields = value === 'empty' || value === 'notempty';
+            case 'logic':
+              var hideValueFields = ['empty', 'notempty', 'between'].indexOf(value) !== -1;
+              var isLogicComparison = value === 'between';
 
-            $selector.find('.panel-title-text .value, #value-dash, #filter-value-type').toggleClass('hidden', hideValueFields);
-            $selector.find('.panel-title-text .value, #value-dash, #filter-value').toggleClass('hidden', hideValueFields);
-          }
+              $selector.find('.panel-title-text .value, #value-dash, #filter-value-type').toggleClass('hidden', hideValueFields);
+              $selector.find('.panel-title-text .value, #value-dash, #filter-value').toggleClass('hidden', hideValueFields);
+
+              $selector.find('.panel-title-text .value, #value-dash, #logic-comparison').toggleClass('hidden', !isLogicComparison);
```

Comment:
```js
$selector.find('.panel-title-text .value, #value-dash, #logic-comparison').toggleClass('hidden', !isLogicComparison);
```

The `hidden` class for `.panel-title-text .value, #value-dash` is toggled based on `hideValueFields` and then `!isLogicComparison`.

If the class needs to be toggled based on a combination of both logic, e.g. `hideValueFields && !isLogicComparison` then please do this only once. Or, if it's only toggled based on `!isLogicComparison` then remove the selector in the previous lines.

Reply:
![изображение](https://user-images.githubusercontent.com/71251809/103348690-70f28a00-4aa3-11eb-95ec-98c270402805.png)

when doing the task was based on this structure because it already existed